### PR TITLE
performed code-cleanup

### PR DESCRIPTION
**Pull request by ioanasechel**

### CODE REVIEW

File: src/main/java/com/sintern/api/DomainController.java [hunk 1]
```diff
@@ -24,6 +24,7 @@ public DomainController(DomainService domainService) {
 
     @GetMapping
     public List<Domain> getDomains(){
+        System.out.println(domainService.getDomains());
```
Comment (DascaluCosmin) [inline src/main/java/com/sintern/api/DomainController.java]: Delete this. If you really want, we can add Loggers.

**Review by LarisaBothaza — APPROVED** [review]

File: src/main/java/com/sintern/api/DomainController.java (hunk 1 above)
Comment (ioanasechel) [reply src/main/java/com/sintern/api/DomainController.java]: My bad :D I accidentally left it there

File: src/main/java/com/sintern/domain/dto/OpenInternPositionDTO.java [hunk 2]
```diff
@@ -1,9 +1,7 @@
-package com.sintern.domain;
+package com.sintern.domain.dto;
 
-import com.fasterxml.jackson.annotation.JsonBackReference;
+import com.sintern.domain.enums.DomainType;
```
Comment (meOtniel) [inline src/main/java/com/sintern/domain/dto/OpenInternPositionDTO.java:4]: Please remove empty spaces between imports

File: src/main/java/com/sintern/domain/entity/EducationDetails.java [hunk 3]
```diff
@@ -1,6 +1,9 @@
-package com.sintern.domain;
+package com.sintern.domain.entity;
```
Comment (meOtniel) [inline src/main/java/com/sintern/domain/entity/EducationDetails.java:2]: Please remove empty space

File: src/main/java/com/sintern/domain/entity/OpenInternPosition.java [hunk 4]
```diff
@@ -1,7 +1,8 @@
-package com.sintern.domain;
+package com.sintern.domain.entity;
```
Comment (meOtniel) [inline src/main/java/com/sintern/domain/entity/OpenInternPosition.java:2]: Please remove empty space

**Review by meOtniel — COMMENTED** [review]
Clean-up looks fine. Good job!

**Review by BrustureanElena — APPROVED** [review]
All good, Ioana :)

**Review by DascaluCosmin — APPROVED** [review]

**Pull request merged**